src/backend/metal/src/device.rs

```diff
@@ -125,6 +125,7 @@ pub struct Device {
     pub(crate) shared: Arc<Shared>,
     pub(crate) private_caps: PrivateCapabilities,
     memory_types: [hal::MemoryType; 4],
+    debug_min_descriptor_count: usize,
```

Comment:
Was this left in this PR intentionally?

Reply:
oh yes, this is an unfortunate workaround we have to get until Dota is fixed